modules/albo-deleting.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: PROCEDURE
+[id="aws-load-balancer-operator-deleting_{context}"]
+= Deleting an AWS Load Balancer Operator installation
```

Comment:
Given that this deletes the "hello world" project, should this be "Deleting the example AWS Load Balancer Operator installation"? Or should it be modified to discuss the general case?